src/schema.ts

```diff
@@ -27,6 +46,17 @@ export interface FieldSchema {
 }
 
 /**
+ * Table Schema
+ * see: https://specs.frictionlessdata.io/table-schema/
+ */
+export interface TableSchema {
```

Comment:
i think we have figured this out below - but just to clarify - `_fieldSchemas` is not redundant - it is just implementing the interface in a particular way.